gems/sorbet-runtime/lib/types/types/attached_class.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+# typed: true
+
+module T::Types
+  # Modeling AttachedClass properly at runtime would require additional
+  # tracking, so at runtime we permit all values and rely on the static checker.
+  class AttachedClassType < Base
+
+    def initialize(); end
+
+    # @override Base
+    def name
+      "AttachedClass"
+    end
+
+    # @override Base
+    def valid?(obj)
+      true
+    end
+
+    # @override Base
+    private def subtype_of_single?(other)
+      case other
+      when AttachedClassType
+          true
+        else
+          false
+      end
+    end
+  end
+end
+
+AttachedClass = T.unsafe(T::Types::AttachedClassType.new)
```

Comment:
What's the `T.unsafe` for?

Reply:
Type-checking the runtime complains that AttachedClass is being re-defined at a different type without it, as it's expected to be a type member with an upper bound of `<root>`.

Reply:
Can you add that as a comment?